autoload/SingleCompile/templates/haskell.vim

```diff
@@ -18,12 +18,28 @@
 " check doc/SingleCompile.txt for more information
 
 function! SingleCompile#templates#haskell#Initialize()
+    " stack-runhaskell
+    call SingleCompile#SetCompilerTemplate('haskell', 'stack-runhaskell',
+               \'The Haskell Tool Stack (stack exec -- runhaskell)', 'stack', 'exec -- runhaskell', '')
+    call SingleCompile#SetPriority('haskell', 'stack-runhaskell', 10)
```

Comment:
Can you explain why you give "stack" series a higher priority? I haven't touched haskell for a long time and I couldn't find a comparison. Is it more popular now?

Reply:
Yes, stack is more popular, and it can manager different version of GHC automatically.